about.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
This injected a special character. I would revert this, but perhaps some systems won't notice it.